### PR TITLE
[10.0] Add hr_recruitment_skill

### PR DESCRIPTION
**Pull request by leemannd**
Add the possibility to add skill for recruitment

Depends on ~~https://github.com/OCA/hr/pull/300~~ https://github.com/OCA/hr/pull/378

### CODE REVIEW

**Comment by gurneyalex** [comment]
@leemannd can you rebase this PR now that #378 is merged, so we get a green runbot?

File: hr_recruitment_skill/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Employee Skills
```
Comment (Matiar) [inline hr_recruitment_skill/README.rst]: Could you please rename the "Employee Skills"--> "Job Skills" or "Applicant Skills" ?

**Review by Matiar — CHANGES_REQUESTED** [review]

File: hr_recruitment_skill/README.rst (hunk 1 above)
Comment (leemannd) [reply hr_recruitment_skill/README.rst]: Hello, or maybe I should follow the technical name of the module with something like "HR recruitment skill'"

Comment (Matiar) [reply hr_recruitment_skill/README.rst]: "Recruitment Skill" is also good to me.

**Comment by leemannd** [comment]
@Matiar Updated. Is it good for you? Do you have it tested?

**Review by Matiar — APPROVED** [review]
Thanks @leemannd , LGTM

**Review by feketemihai — APPROVED** [review]
Not tested...

File: hr_recruitment_skill/models/hr_job.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+from odoo import fields, models
+
+
+class HrJob(models.Model):
+    _inherit = 'hr.job'
+
+    required_skill_ids = fields.Many2many(
+        'hr.skill',
+        relation='hr_jobs_required_skills',
```
Comment (guewen) [inline hr_recruitment_skill/models/hr_job.py:13]: Are you sure about this relation's name? I couldn't find it in the dependencies.

File: hr_recruitment_skill/i18n/fr.po [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: hr_recruitment_skill
+#: model:ir.model,name:hr_recruitment_skill.model_hr_job
+msgid "Job Position"
+msgstr "Poste Occupé"
```
Comment (guewen) [inline hr_recruitment_skill/i18n/fr.po:21]: In French usually we don't capitalize every word as we do in English

**Pull request merged**